### PR TITLE
New ESLZ graph queries added

### PR DESCRIPTION
**Pull request by fskelly**
New queries with only the JSON commited.

### CODE REVIEW

File: checklists/lz_checklist.en.json [hunk 1]
```diff
@@ -432,7 +438,7 @@
     {
       "category": "Networking",
       "subcategory": "Virtual WAN",
-      "text": "Ensure that the network architecture is within the Azure Virtual WAN limits.",
+      "text": "Ensure that the network architecture is within the?Azure Virtual WAN limits.",
```
Comment (erjosito) [inline checklists/lz_checklist.en.json:441]: Could you please remove the `?` here?

File: checklists/lz_checklist.en.json [hunk 2]
```diff
@@ -1021,7 +1027,7 @@
     {
       "category": "Management",
       "subcategory": "Monitoring",
-      "text": "Use Update Management in Azure Automation as a long-term patching mechanism for both Windows and Linux VMs. ",
+      "text": "Use Update Management in Azure Automation?as a long-term patching mechanism for both Windows and Linux VMs. ",
```
Comment (erjosito) [inline checklists/lz_checklist.en.json:1030]: Same thing, could you please remove the `?` here?

File: checklists/lz_checklist.en.json [hunk 3]
```diff
@@ -1207,11 +1213,19 @@
     {
       "category": "Security",
       "subcategory": "Secrets",
-      "text": "Use a vault per application per environment per region.",
+      "text": "Don't share Key Vault instances between applications to avoid secret sharing across environments.",
```
Comment (erjosito) [inline checklists/lz_checklist.en.json:1216]: Did you change this on purpose? I think Charles changed this checklist item over the weekend, you might have an old version of it...

File: checklists/lz_checklist.en.json [hunk 4]
```diff
@@ -1207,11 +1213,19 @@
...
       "link": "https://docs.microsoft.com/azure/key-vault/general/best-practices"
     },
+    {
+      "category": "Security",
+      "subcategory": "Secrets",
+      "text": "Don't use centralized instances of Key Vault for application keys or secrets.",
+      "guid": "43276d8b-dc05-45bc-b619-e8a13f988795",
+      "severity": "Medium",
+      "link": "https://docs.microsoft.com/azure/key-vault/general/best-practices"
+    },
```
Comment (erjosito) [inline checklists/lz_checklist.en.json:1228]: Same here, I think this was removed over in a recent PR, since it is redundant with the previous check.

**Review by erjosito — COMMENTED** [review]
Sorry to keep putting obstacles, but it looks like there are some conflicts with some recent PRs. If you could fix those, it would be great. Thanks!

File: checklists/lz_checklist.en.json (hunk 1 above)
Comment (fskelly) [reply checklists/lz_checklist.en.json:441]: Fixed :)

File: checklists/lz_checklist.en.json (hunk 4 above)
Comment (fskelly) [reply checklists/lz_checklist.en.json:1228]: removed whole check.

File: checklists/lz_checklist.en.json (hunk 2 above)
Comment (fskelly) [reply checklists/lz_checklist.en.json:1030]: fixed

**Review by erjosito — APPROVED** [review]

**Comment by erjosito** [comment]
Oops, I see there were some JSON lint checks?

**Comment by fskelly** [comment]
It is part of the regex for the Resource Graph query....

resources 
| where type == 'microsoft.network/virtualnetworks' 
| extend addressSpace = todynamic(properties.addressSpace) 
| extend addressPrefix = todynamic(properties.addressSpace.addressPrefixes) 
| mvexpand addressSpace 
| mvexpand addressPrefix 
| project name, location, resourceGroup, subscriptionId, cidr = addressPrefix 
| where not (cidr matches regex @'^(?:10|127|172\.(?:1[6-9]|2[0-9]|3[01])|192\.168)\..*')

**Comment by erjosito** [comment]
Ah, I think it is the backslash characters at the end of the query. I think you need to replace those `\` with `\\`?

**Comment by fskelly** [comment]
Just confirming that if i do that - the actual query will remain in place. If i change the ARG to have the \\ and NOT \, the actual result is different

**Review by erjosito — APPROVED** [review]

**Pull request merged**